dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/S3Utils.java

```diff
@@ -175,7 +176,21 @@ public String getFileName(ResourceType resourceType, String tenantCode, String f
     }
 
     @Override
-    public void download(String tenantCode, String srcFilePath, String dstFile, boolean deleteSource, boolean overwrite) throws IOException {
+    public void download(String tenantCode, String srcFilePath, String dstFilePath, boolean deleteSource, boolean overwrite) throws IOException {
+        File dstFile = new File(dstFilePath);
+        if (dstFile.exists()) {
+            if (dstFile.isFile()) {
+                if (overwrite) {
+                    Files.delete(dstFile.toPath());
```

Comment:
this isn't needed, FileOutputStream will overwrite default

Reply:
yes, You are right, I'll modify it